smoke-test/src/test/java/org/opennms/smoketest/minion/TelemetryIT.java

```diff
@@ -0,0 +1,340 @@
+                .eq("eventUei", EventConstants.NEW_SUSPECT_INTERFACE_EVENT_UEI).ge("eventTime", startOfTest)
+                .eq("ipAddr", Inet4Address.getByName(SENDER_IP_MINION)).toCriteria();
+
+        await().atMost(1, MINUTES).pollInterval(10, SECONDS).until(DaoUtils.countMatchingCallable(eventDao, criteria),
+                greaterThan(0));
+
+        final OnmsNode onmsNode = await().atMost(1, MINUTES).pollInterval(5, SECONDS)
+                .until(DaoUtils.findMatchingCallable(nodeDao,
+                        new CriteriaBuilder(OnmsNode.class).eq("label", SENDER_IP_MINION).toCriteria()),
+                notNullValue());
+
+        assertThat(onmsNode.getLocation().getLocationName(), is("MINION"));
+
+        LOG.info(" New suspect event has been sent and node has been created for IP : {}", SENDER_IP_MINION);
+        final InetSocketAddress minionUdp = m_testEnvironment.getServiceAddress(ContainerAlias.MINION, 50000, "udp");
+
+        TelemetryTop.TelemetryStream jtiMsg = buildJtiMessage(SENDER_IP_MINION, "eth0_system_test", 100, 100);
+        byte[] jtiMsgBytes = jtiMsg.toByteArray();
+        DatagramPacket packet = new DatagramPacket(jtiMsgBytes, jtiMsgBytes.length, minionUdp);
+
+        try (DatagramSocket socket = new DatagramSocket()) {
+            socket.send(packet);
+        } catch (IOException e) {
+            LOG.error("Exception while sending jti packets", e);
+        }
+        await().atMost(2, MINUTES).pollDelay(0, SECONDS).pollInterval(15, SECONDS)
+                .until(matchRrdFileFromNodeResource(opennmsHttp, executor, onmsNode.getId()));
+        matchRrdFileFromNodeResource(opennmsHttp, executor, onmsNode.getId());
+    }
+
+    public static Callable<Boolean> matchRrdFileFromNodeResource(InetSocketAddress opennmsHttp, Executor executor,
+            Integer id) throws ClientProtocolException, IOException {
+        return new Callable<Boolean>() {
+
+            @Override
+            public Boolean call() throws Exception {
+                HttpResponse response = executor
+                        .execute(Request.Get(String.format("http://%s:%d/opennms/rest/resources/fornode/%d",
+                                opennmsHttp.getAddress().getHostAddress(), opennmsHttp.getPort(), id)))
+                        .returnResponse();
+
+                String message = EntityUtils.toString(response.getEntity());
+                System.out.println(message);
+                if (message.contains("rrdFile=\"ifOutOctets") && message.contains("eth0_system_test")) {
+                    return true;
+                } else {
+                    return false;
+                }
+
+            }
+        };
+    }
+
+    private static TelemetryTop.TelemetryStream buildJtiMessage(String ipAddress, String ifName, long ifInOctets,
+            long ifOutOctets) {
+        final Port.GPort port = Port.GPort.newBuilder()
+                .addInterfaceStats(
```

Comment:
The indentation in this block make this unreadable. Compare this to https://github.com/OpenNMS/opennms/blob/f1d252b5bc0b9a6bcf58ed9dbaa89bc3ec09aa4a/features/telemetry/adapters/jti/src/main/java/org/opennms/netmgt/telemetry/adapters/jti/JtiClient.java#L54.